my-react-app/src/app/screens/Game/components/Board/index.js

```diff
@@ -29,4 +30,9 @@ class Board extends React.Component {
   }
 }
 
+Board.propTypes = {
+  squares: PropTypes.array,
```

Comment:
We don´t use `PropTypes.array`. Use `PropTypes.arrayOf(thePropTypeOfTheElementOfTheArray)`.

e.g. If you are passing an array of numbers, use `PropTypes.arrayOf(PropTypes.string)`.
Also, it´s required for this component to work